guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -30,7 +30,6 @@ You can use future-dated subscriptions in a subscription allocation.
 When you add future-dated subscriptions to content hosts before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
 Manually attach the future-dated subscriptions to your content hosts before the current subscriptions expire.
-Do not rely on the auto-attach method because this method is designed for a different purpose and might not work.
```

Comment:
I remember seeing a Google doc covering this part; I think those modifications should be applied here.